app/controllers/categories_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CategoriesController < ApplicationController
   def index
-    @categories = Category.all
+    @projects = Project.all.reject(&:retired)
```

Comment:
This line of code gets changed in another pull request (the paperclip life) as per Mejia's advice. Should I merge this in anyway fully knowing that it gets changed in a later PR?